src/components/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Tab } from '../react-app-env';
+import './Tabs.scss';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <>
+      {tabs.map(tab => (
+        <div
+          className="container"
+          key={tab.id}
+        >
+          <button
+            className="button"
+            type="button"
+            onClick={() => {
+              onTabSelected(tab);
```

Comment:
Will be good to call this function only if tab id is not equal to selected tab id